Fix bug #672100

presage: FTBFS: profileManagerTest.cpp:86:47: error: 'unlink' was not declared in this scope
--- a/test/lib/core/profileManagerTest.cpp
+++ b/test/lib/core/profileManagerTest.cpp
@@ -32,6 +32,11 @@
 
 #include <fstream>
 
+#ifdef HAVE_UNISTD_H
+# include <unistd.h>
+#endif
+
+
 CPPUNIT_TEST_SUITE_REGISTRATION( ProfileManagerTest );
 
 void ProfileManagerTest::setUp()
--- a/test/lib/predictors/smoothedNgramPredictorTest.cpp
+++ b/test/lib/predictors/smoothedNgramPredictorTest.cpp
@@ -22,12 +22,14 @@
 									     **********(*)*/
 
 
+
+#include "smoothedNgramPredictorTest.h"
+
 // for unlink(), getcwd()
 #ifdef HAVE_UNISTD_H
 # include <unistd.h>
 #endif
 
-#include "smoothedNgramPredictorTest.h"
 
 CPPUNIT_TEST_SUITE_REGISTRATION( SmoothedNgramPredictorTest );
 
